src/heroku/orb.yml

```diff
@@ -1,35 +1,30 @@
 version: 2
 description: "Provides basic functionality for deploying to Heroku"
 
-commands:
-  install:
+executors:
+  default:
     parameters:
-      os:
-        default: “linux”
-        description: The target Operating System for the heroku binary. Must be one of "linux", “darwin”, “win32”.
-        type: string
-        # enum types are not yet implemented.
-        # type: enum
-        # enum: ["linux", “darwin”, “win32”]
-      arch:
+      tag:
         type: string
-        default: "x86"
-        description: The target CPU architecture for the heroku binary. Must be one of “x64”, “x86”, “arm”.
-        # enum ypes are not yet implemented
-        # type: enum
-        # enum: [“x64”, “x86”, “arm”]
+        default: "bionic"
+    docker:
+      - image: buildpack-deps:<< parameters.tag >>
 
+commands:
+  install:
     steps:
       - run:
           name: "Install Heroku CLI, if necessary"
-          # TODO: the assumption of wget, etc. below likely wouldn't work in all cases, especially windows.
           command: |
             if [[ $(command -v heroku) == "" ]]; then
-              wget https://cli-assets.heroku.com/heroku-cli/channels/stable/heroku-cli-{{parameters.os}}-{{parameters.arch}}.tar.gz -O heroku.tar.gz
-              mkdir $HOME/heroku
-              tar -xvzf heroku.tar.gz -C $HOME/heroku --strip-components 1
-              cd $HOME/heroku
-              ./install
+              curl https://cli-assets.heroku.com/install.sh | sh
             else
               echo "Heroku is already installed. No operation was performed."
             fi
+
+  deploy:
+    steps:
+      - run:
+          name: Deploy Master to Heroku
+          command: |
+            git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
```

Comment:
We should make the branch name here a parameter that can default to master (or be required )